config/seeds.js

```diff
@@ -38,6 +38,8 @@ const contractCalls = [
   ['Contribution', 'add', [{ contributorId: 1, contributorIpfsHash: 'QmWKCYGr2rSf6abUPaTYqf98urvoZxGrb7dbspFZA6oyVF', date: '2019-04-11', amount: 5000, kind: 'dev', description: '[67P/kredits-contracts] Introduce contribution token', url: '' }, { gasLimit: 350000 }]],
   ['Contribution', 'add', [{ contributorId: 2, contributorIpfsHash: 'QmcHzEeAM26HV2zHTf5HnZrCtCtGdEccL5kUtDakAB7ozB', date: '2019-04-11', amount: 1500, kind: 'dev', description: '[67P/kredits-web] Reviewed stuff', url: '' }, { gasLimit: 350000 }]],
   ['Contribution', 'claim', [1, { gasLimit: 300000 }]],
+
+  ['Reimbursement', 'add', [{title: 'Server Hosting', description: 'All the serverz', amount: 1, token: '0xa3048576e296207eb0141f2803590ad044f81928', currency: 'WBTC', date: '2020-05-28'}, { gasLimit: 300000 }]],
```

Comment:
This seems inconsistent with the expense schema. I don't see how it creates the array of expenses, separate from the WBTC reimbursement. Would also be good to add two expenses for an example reimbursement from the start (ideally just the same example as in the schemas repo) for UI development.

Reply:
the array here is not an array of expenses but just how it is passed to the `add` function. see above the seeds for contributions. 

The data here is not correct yet. and yep, we can add more just by c&p this line.

Reply:
> the array here is not an array of expenses but just how it is passed to the `add` function. see above the seeds for contributions.

Yes, that's not what I meant. I meant that a reimbursement is an array of expenses, so it needs be an array for the `add` function and the examples and seeds as well.